src/variables/actions/thunks.ts

```diff
@@ -267,26 +267,26 @@ export const createVariable = (
   }
 }
 
-export const createVariableFromTemplate = (
-  template: VariableTemplate
-) => async (dispatch: Dispatch<Action>, getState: GetState) => {
+export const createVariableFromTemplate = (template: api.Template) => async (
+  dispatch: Dispatch<Action>,
+  getState: GetState
+) => {
   try {
     const state = getState()
     const org = getOrg(state)
-    const resp = await createVariableFromTemplateAJAX(template, org.id)
+    // only one variable in the exported template so we can safely just index the 0th element
+    const variableName = template[0].spec.name
+
+    await createResourceFromPkgerTemplate(template, org.id)
+
+    await getVariablesAction()(dispatch, getState)
 
-    const createdVar = normalize<Variable, VariableEntities, string>(
-      resp,
-      variableSchema
-    )
     event('variable.create.from_template.success', {
-      id: resp?.id,
-      name: resp?.name,
+      name: variableName,
     })
-    dispatch(setVariable(resp.id, RemoteDataState.Done, createdVar))
-    dispatch(notify(copy.createVariableSuccess(resp.name)))
+    dispatch(notify(copy.createVariableSuccess(variableName)))
   } catch (error) {
-    event('variable.create.from_template.failure', {template: template?.id})
+    event('variable.create.from_template.failure', {template: template[0].meta.name})
```

Comment:
Using the template name instead of ID. The pkgr response doesn't seem to have an ID.